<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerProtonDoseModule</class>
 <widget class="qSlicerWidget" name="qSlicerProtonDoseModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProtonDose</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_ParameterSetNode">
       <property name="text">
        <string>Parameter set:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ParameterSet">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLProtonDoseNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_CTInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CT Input</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_InputImage">
        <property name="text">
         <string>Input image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_inputImage">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_BeamParameters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Beam Parameters</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_Beam">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Beam:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Beam">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>1: AP</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_changeName">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Change name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AddBeam">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Add Beam</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="GroupBox_isocentre">
           <property name="title">
            <string>Isocentre</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_isoX">
                <property name="text">
                 <string>ISO X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_IsoX">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_IsoY">
                <property name="text">
                 <string>ISO Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_IsoY">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_IsoZ">
                <property name="text">
                 <string>ISO Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_IsoZ">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_BeamGeometry">
           <property name="title">
            <string>Beam Geometry</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_Gantry">
                <property name="text">
                 <string>Gantry</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_Gantry">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_Range">
                <property name="text">
                 <string>          Range</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLineEdit" name="lineEdit_Range">
                <property name="maximumSize">
                 <size>
                  <width>133</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLabel" name="label_Mod">
                <property name="text">
                 <string>             Mod</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QLineEdit" name="lineEdit_Mod">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_Collimator">
                <property name="text">
                 <string>Collimator</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_Collimator">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_XJaw">
                <property name="text">
                 <string>X Jaw</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLineEdit" name="lineEdit_XJaw">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLabel" name="label_YJaw">
                <property name="text">
                 <string>         Y Jaw</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="7">
               <widget class="QLineEdit" name="lineEdit_YJaw">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="5">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Couch</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>18</number>
              </property>
              <item>
               <widget class="QLabel" name="label_CouchRotation">
                <property name="text">
                 <string>Rotation</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_CouchRotation">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_CouchPitch">
                <property name="text">
                 <string>Pitch</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_CouchPitch">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_CouchRoll">
                <property name="text">
                 <string>Roll</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_CouchRoll">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-180.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>180.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_DeleteBeam">
             <property name="text">
              <string>Delete Beam</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_LoadBeams">
             <property name="text">
              <string>Load Beams</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_SaveBeams">
             <property name="text">
              <string>Save Beams</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder>GroupBox_isocentre</zorder>
        <zorder>groupBox_BeamGeometry</zorder>
        <zorder>groupBox</zorder>
        <zorder></zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_CommentGeometry">
     <property name="text">
      <string>Geometry uses IEC 61217 convention</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_BeamModel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Beam Model</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_MinEnergy">
        <property name="text">
         <string>Min Energy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_MinEnergy"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_EnergySpread">
        <property name="text">
         <string>Energy Spread</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_EnergySpread"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_MaxEnergy">
        <property name="text">
         <string>Max Energy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_MaxEnergy"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_EnergyDelta">
        <property name="text">
         <string>Energy Delta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_EnergyDelta"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Output">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dose Output</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_OutputImage">
        <property name="text">
         <string>Output image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_OutputHierarchy">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CollapsibleButton_BeamParameters</tabstop>
  <tabstop>CollapsibleButton_BeamModel</tabstop>
  <tabstop>CollapsibleButton_Output</tabstop>
  <tabstop>lineEdit_MinEnergy</tabstop>
  <tabstop>lineEdit_EnergySpread</tabstop>
  <tabstop>lineEdit_MaxEnergy</tabstop>
  <tabstop>lineEdit_EnergyDelta</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerProtonDoseModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ParameterSet</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProtonDoseModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_OutputHierarchy</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>2</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
